api/db/migration.go

```diff
@@ -16,9 +16,12 @@ func MigrateUp(directory, environment, schema string) error {
 		return err
 	}
 
-	target, err := goose.GetMostRecentDBVersion(conf.MigrationsDir)
+	target, err := goose.NumericComponent(os.Getenv("DB_MIGRATION_TARGET"))
```

Comment:
This allows us to target a specific migration version (up or down) via the environment variable.